integration-tests/src/test/java/oracle/kubernetes/operator/ItOperatorUpgrade.java

```diff
@@ -89,145 +104,33 @@ public void cleanupOperatorAndDomain() throws Exception {
         operator.destroy();
       }
       TestUtils.exec("rm -rf " + Paths.get(opUpgradeTmpDir).toString());
+      TestUtils.exec("kubectl delete crd domains.weblogic.oracle --ignore-not-found");
```

Comment:
should check this actually worked by trying to get the crd again

Reply:
This is the cleanup section, when we delete the CRD from default NameSpace after the test is completed, so that it will not interfere with other test

Reply:
CRD's are not namespaced, they are at the cluster scope.  So deleting a CRD by definition would almost certainly interfere with other tests.
The original question stands though - you are no checking that it was deleted.

Reply:
Upgrade test will not be running in parallel with other Tests, so deleting a CRD will not affect any other tests.  I have added an Assertion to make sure CRD is deleted completely form K8s. 

TestUtils.exec("kubectl delete crd domains.weblogic.oracle --ignore-not-found");
// Make sure domain CRD is deleted form k8s 
ExecResult result = TestUtils.exec("kubectl get crd domains.weblogic.oracle");
Assertions.assertEquals(1, result.exitValue());